docs/atlascli/command/atlas-clusters-failover.txt

```diff
@@ -0,0 +1,97 @@
+.. _atlas-clusters-failover:
+
+=======================
+atlas clusters failover
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Starts a failover test for the specified cluster in the specified project.
+
+Clusters contain a group of hosts that maintain the same data set. A failover test checks how MongoDB Cloud handles the failure of the cluster's primary node. During the test, MongoDB Cloud shuts down the primary node and elects a new primary.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas clusters failover <clusterName> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterName
+     - string
+     - true
+     - Name of the cluster to start a failover test.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --force
+     - 
+     - false
+     - If specified, skips asking for confirmation before proceeding with a requested action.
+   * - -h, --help
+     - 
+     - false
+     - help for failover
+   * - --projectId
+     - string
+     - false
+     - Project ID to use. Overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-save-connection-settings/. To learn about profiles for MongoCLI, see https://www.mongodb.com/docs/mongocli/stable/configure/configuration-file/.
+
```

Comment:
For links to the product documentation, request someone from the Docs team create persistent dochub URLs. I created a couple for these.

```suggestion
     - Human-readable label that identifies the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings. To learn about profiles for MongoCLI, see https://dochub.mongodb.org/core/atlas-cli-configuration-file.

```